src/helpers/api.js

```diff
@@ -99,7 +99,7 @@ class Api {
       fn: this._getTrial,
       params: trialId,
       validate: trial => trial.completed_at !== undefined,
-      // maxAttempts: 10,
+      // maxAttempts: 10,  // This should be on, but it totally breaks the page currently
```

Comment:
Created a separate ticket to address these issues. This existed prior to the React upgrade, but the upgrade broke Back Button functionality, and the now-commented-out lines were related to the issue, but fixing this entirely was out of the scope of this task.